tests/test_camera_readout.py

```diff
@@ -73,6 +75,51 @@ def test_get_amp_image(self):
         self.assertTupleEqual(image.getArray().shape, (2020, 532))
 
 
+class NoaoKeywordTestCase(unittest.TestCase):
+    "TestCase class for raft-level NOAO mosaicking keywords"
```

Comment:
Does  "mosaicking" refer to "NOAO Mosaic Camera" keywords as in 
http://iraf.noao.edu/projects/ccdmosaic/imagedef/mosaic/MosaicV1.html ?

or keywords related to making mosaics?  The code looks like it is  checking all keywords exists as in TS8 data.  Perhaps "mosaicking" should be left off? I'm not familiar enough to know.

Reply:
It's not checking all of the keywords--the ones in the `exclude` tuple are being excluded.  The ones remaining in those harvested headers all pertain to mosaicking.